### PR TITLE
findGroups

### PR DESCRIPTION
**Pull request by Danduu**
h1. I ensured:
* [x] I've updated the unit tests and covered enough parts of the code (`gulp lint` and `gulp test`)
* [x] I've updated the API documentation within the code and generated a fresh export (`gulp docs`)

Contains 📦 

* [x] Versioning
* [x] added findgroups method to the adapter
* [x] Updated unit tests and refactored HatAdapter to GibbonAdapter in unit tests

### CODE REVIEW

File: test/gibbon-adapter.test.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 const expect = require('chai').expect;
 const helper = require('./helper');
 
-const HatAdapter = require('../index').GibbonAdapter;
+const GibbonAdapter = require('../index').GibbonAdapter;
```
Comment (ghost) [inline test/gibbon-adapter.test.js:5]: Thanks! Missed that one :-O

**Comment by ghost** [comment]
Thanks @Danduu !

I think, when adding your suggestion `findGroups`, having a `findPermissions`and a `findUsers` is also nice to have right? ;)

I know you've ensured your coverage (LokiJS adapter is ignored ;-) ), but I dearly would like to also have 1 or 2 unit tests with examples in the `LokiJSGibbonAdapter` if possible (See: `gibbons.test.js`).

It this possible?

**Comment by ghost** [comment]
@Danduu You are the first one to participate on this small project, you are forever in my debt ^^
Keep up the good work!

**Comment by icazemier** [comment]
Hi @NDamhuis / @Danduu ,

I Hope you're doing well 😊.

Closing this one.

(I even removed the whole adapter approach, long time ago)

Regards,

Ivo

**Pull request closed without merging**